src/compiler/diagnosticMessages.json

```
@@ -879,6 +879,10 @@
         "category": "Error",
         "code": 1266
     },
+    "Property '{0}' cannot have an initializer because it is marked abstract.": {

```

```suggestion
    "Property '{0}' cannot have an initializer because it is marked 'abstract'.": {
```

Maybe? Then https://github.com/microsoft/TypeScript/actions?query=workflow%3A%22Accept+Baselines+and+Fix+Lints%22